Example/IPDecimalNumberPad/Controllers/IPViewController.m

```diff
@@ -0,0 +1,76 @@
+//
+//  IPViewController.m
+//  IPDecimalNumberPad
+//
+//  Created by Nick Servidio on 10/23/2015.
+//  Copyright (c) 2015 Nick Servidio. All rights reserved.
+//
+
+#import "IPViewController.h"
+#import "IPDecimalNumberPad.h"
+#import "CAGradientLayer+IPGradients.h"
+#import "IPStringBackedDecimalValue.h"
+
+@interface IPViewController () <IPDecimalNumberPadDelegate>
+
+@property (weak, nonatomic) IBOutlet UILabel *instructionLabel;
+@property (weak, nonatomic) IBOutlet UILabel *numberLabel;
+@property (weak, nonatomic) IBOutlet IPDecimalNumberPad *numberPad;
+@property (strong, nonatomic) CAGradientLayer *backgroundLayer;
+@property (strong, nonatomic) IPStringBackedDecimalValue *currentAmount;
+
+@end
+
+@implementation IPViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self setupBackgroundGradient];
+    [self setupLabels];
+    self.numberPad.delegate = self;
+    self.currentAmount = [[IPStringBackedDecimalValue alloc] init];
+}
+
+- (void)viewDidLayoutSubviews {
+    [super viewDidLayoutSubviews];
+    self.backgroundLayer.frame = self.view.bounds;
```

Comment:
Is there a reason not to use constraints here? this would make this most up to date with our current coding standards